common/src/main/java/com/wynntils/models/players/type/GuildRank.java

```diff
@@ -5,12 +5,12 @@
 package com.wynntils.models.players.type;
 
 public enum GuildRank {
-    RECRUIT("Recruit", ""),
-    RECRUITER("Recruiter", "★"),
-    CAPTAIN("Captain", "★★"),
-    STRATEGIST("Strategist", "★★★"),
+    OWNER("Owner", "★★★★★"),
     CHIEF("Chief", "★★★★"),
-    OWNER("Owner", "★★★★★");
+    STRATEGIST("Strategist", "★★★"),
+    CAPTAIN("Captain", "★★"),
+    RECRUITER("Recruiter", "★"),
+    RECRUIT("Recruit", "");
```

Comment:
Was there a reason for this?

Reply:
Ah good thing you asked, I thought I checked everywhere but I missed `GuildRank.values()[stars]` in the replacement feature. Will just have to reverse the list then